examples/put_to_self.c

```diff
@@ -25,31 +25,32 @@
 
 /*
  * This example show the sending of a message to ourselves, we get our Id,
- * create a message and send it through a specified allocated buffer"
+ * create a message and send the message to our Id"

```

Comment:
Would propose something a bit more detailed.
"This example shows a PUT data transfer to ourselves:
- prepare target Portals resources: Event Queue, Portals Table Entry and an List Entry covering the receive buffer
- prepare initiator Portals resources: Event Queue (same than target), Memory Descriptor covering message buffer.
- data transfer with PUT operation and wait for completion events"